bot/cogs/moderation/scheduler.py

```diff
@@ -325,7 +326,7 @@ def mod_log(self) -> ModLog:
         log_text = {
             "Member": f"<@{user_id}>",
             "Actor": str(self.bot.get_user(actor) or actor),
-            "Reason": infraction["reason"],
+            "Reason": textwrap.shorten(infraction["reason"], width=1500, placeholder="..."),
```

Comment:
This is not ideal but I recognise this case is more troublesome than others. Do you think there is a simple way to ensure the reason will be at the end given this gets merged with another dict may have additional keys added by `pardon_infraction`? I suspect it will involve having to delete the reason from the dict then add it back in. Maybe just not worth the trouble.

Reply:
Maybe should this overwrite `Reason` in `pardon_infraction` to put this as the last key? And after this, `textwrap.shorten` whole ModLog embed description?